scripts/ecs/appsignals/setup-ecs-demo.sh

```diff
@@ -0,0 +1,1124 @@
+#!/bin/bash
+set -ex
+
+# Default values
+DEFAULT_REGION="us-east-1"
+DEFAULT_CLUSTER="ecs-pet-clinic-demo"
+OPERATION="create"
+
+# Read command line arguments
+for i in "$@"
+do
+case $i in
+    --operation=*)
+    OPERATION="${i#*=}"
+    shift # past argument=value
+    ;;
+    --region=*)
+    REGION="${i#*=}"
+    shift # past argument=value
+    ;;
+    --cluster=*)
+    CLUSTER="${i#*=}"
+    shift # past argument=value
+    ;;
+    *)
+          # unknown option
+    ;;
+esac
+done
+
+# Set region cluster name with provided value or default
+REGION="${REGION:-$DEFAULT_REGION}"
+CLUSTER="${CLUSTER:-$DEFAULT_CLUSTER}"
+
+export AWS_DEFAULT_REGION=$REGION
+
+# Variables
+SG_NAME="ecs-security-group"
+IAM_TASK_ROLE_NAME="ecs-pet-clinic-task-role-${REGION}"
+IAM_EXECUTION_ROLE_NAME="ecs-pet-clinic-execution-role-${REGION}"
+LOAD_BALANCER_NAME="ecs-pet-clinic-lb-${REGION}"
+OUTPUT_FILE="ecs-pet-clinic-vars.txt"
+account_id="000111222333"
+
+ACCOUNT_ID=$(aws sts get-caller-identity --query 'Account' --output text)
+VPC_ID="vpc-0000000000aaaaaaa"
```

Comment:
Why do we need these fake values? Can we declare them as empty?

Reply:
Yes, update them to be empty string.